bgpd/bgp_vty.c

```diff
@@ -13177,7 +13177,9 @@ static void bgp_show_peer(struct vty *vty, struct peer *p, bool use_json,
 		/* BGP Version. */
 		json_object_int_add(json_neigh, "bgpVersion", 4);
 		json_object_string_addf(json_neigh, "remoteRouterId", "%pI4",
-					&p->remote_id);
+					(p->status == Established)
```

Comment:
If we are in openConfirm ( and thus have this data ) why are we not showing it?

Reply:
We are trying to make key fields in JSON output persistent. It should be friendly for script analysis.